harp/reader.py

```diff
@@ -214,7 +236,7 @@ def create_reader(
     keep_type: bool = False,
 ):
     """Creates a device reader object from the specified dataset or schema.
-    
+
```

Comment:
Indeed this was an issue with the code on main, I must have forgotten to run black before the changes to docs (need to setup a pre-commit hook). I have fixed it now.

Can you rebase the PR off the latest main? That should get rid of these style-only changes.